packages/url_launcher/url_launcher_platform_interface/CHANGELOG.md

```diff
@@ -1,8 +1,4 @@
-## 2.0.0-nullsafety.1
-
-* Bump Dart SDK to support null safety.
-
-## 2.0.0-nullsafety
```

Comment:
hmm. do we remove the prereleases notes?

Reply:
That's what I was told: https://github.com/flutter/plugins/pull/3543#discussion_r575427610

We should capture everything that is interesting in aggregate into the stable changelog though, in cases where there are other fixes.